migration/src/main/resources/migrations/ensure-order-type-has-java_class_name-column.xml

```diff
@@ -5,49 +5,121 @@
-            <comment>
-                Updating java class names under order_type table
-            </comment>
-            <update tableName="order_type">
-                <column name="java_class_name" value="org.openmrs.TestOrder"/>
-                <where>name='Lab test'</where>
-            </update>
-            <update tableName="order_type">
-                <column name="java_class_name" value="org.openmrs.DrugOrder"/>
-                <where>name='Drug order'</where>
-            </update>
-            <update tableName="order_type">
-                <column name="java_class_name" value="org.openmrs.LunchOrder"/>
-                <where>name='Lunch order'</where>
-            </update>
-        </changeSet>
-     -->
-        <!--
-            TODO: Ensure all existing order types have a valid java_class_name populated
-            This should include for drug order, since core will only set this if the uuid of this type is 131168f4-15f5-102d-96e4-000c29c2a5d7, and ours is not
-        -->
+    <changeSet id="ensure-zidolamN-order-type-has-java_class_name-value" author="IMB">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT COUNT(*) FROM order_type
+                WHERE name = 'Zidolam-N' AND java_class_name IS NULL
+            </sqlCheck>
+        </preConditions>
+        <comment>Setting java_class_name column for Zidolam-N order type row(Rwink)</comment>
+        <update tableName="order_type">
+            <column name="java_class_name" value="org.openmrs.DrugOrder" />
+            <where>name='Zidolam-N'</where>
+        </update>
     </changeSet>
 
+    <changeSet id="ensure-penicillin-order-type-has-java_class_name-value" author="IMB">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT COUNT(*) FROM order_type
+                WHERE name = 'Penicillin vk' AND java_class_name IS NULL
+            </sqlCheck>
+        </preConditions>
+        <comment>Setting java_class_name column for penicillin order type row(Butaro)</comment>
+        <update tableName="order_type">
+            <column name="java_class_name" value="org.openmrs.DrugOrder" />
+            <where>name='Penicillin vk'</where>
+        </update>
+    </changeSet>
+
+    <changeSet id="ensure-lab-order-type-has-java_class_name-value" author="IMB">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT COUNT(*) FROM order_type
+                WHERE name = 'Lab Order' AND java_class_name IS NULL
+            </sqlCheck>
+        </preConditions>
+        <comment>Setting java_class_name column for Lab Order type row</comment>
+        <update tableName="order_type">
+            <column name="java_class_name" value="org.openmrs.TestOrder" />
+            <where>name='Lab Order'</where>
+        </update>
+    </changeSet>
+
```

Comment:
Another option here for cleaning up.  Are "Lab Order" and "Test" meaningfully different, or is this something we should be looking to harmonize